src/Components/Buttons/SortByAlphabet.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { SortType } from '../../Enums/SortType';
+import cn from 'classnames';
+import { SortingProps } from '../../types/buttons/SortingProps';
+
+export const SortByAlphabet: React.FC<SortingProps> = ({
+  sortBy,
+  setSortBy,
+}) => {
+  return (
+    <button
+      type="button"
+      className={cn('button is-info', {
+        'is-light': sortBy !== SortType.ALPHABET,
+      })}
+      onClick={() => setSortBy(SortType.ALPHABET)}
+    >
+      Sort alphabetically
+    </button>
```

Comment:
[UX] - To improve user experience, consider marking the button as active when the `sortBy` is `SortType.ALPHABET`. You've already applied a different class when it's not the active sort type, but it would be beneficial to visually indicate the active state as well. This could be done by adding another class when `sortBy` is `SortType.ALPHABET`, or changing the text of the button to show it's the current sort order.